lib/api_client.ex

```diff
@@ -1,23 +1,10 @@
 defmodule GlassFactoryApi.ApiClient do
   @moduledoc """
-  Client which make external HTTP Request
+  Client which make external HTTP Request.
   """
 
-  alias GlassFactoryApi.HttpAdapter
-
   @access_info Application.get_env(:glass_factory_api, GlassFactoryApi.ApiClient)
 
-  @type http_response() :: {atom(), HttpAdapter.response()}
-
-  @doc """
-  Make a GET request using the given adapter to retrieve the given resource
-
-  ## Examples
-
-      iex> GlassFactoryApi.ApiClient.get("members")
-      {:ok , %{body: "[]", headers: [], status_code: 200}}
-  """
-  @spec get(String.t()) :: http_response()
```

Comment:
Why all this _juicy_ documentation was removed? 👀

Reply:
My mistake 😅